Responsive drupal.hu

csakiistvan opened the pull request:
A demon sminket alsminké konvertálva, bootstrap alapon megvalósult a respozivitás. A struktúrát és fontosabb dolgokat nem változtattam, igyekeztem átvenni a jelenlegi formázást. Csupán pár dolog volt amit feltétlen át kellett alakítanom (vote up/down és social képek cseréje, és hasonló apróbb dolgok).

A következő parancsok szükségesek a teszteléshez:
- drush updb -y
- drush cc all

csakiistvan committed:
Delete not used style files

csakiistvan committed:
Delete included stylesheets from theme

csakiistvan committed:
Delete not valid package name from theme

csakiistvan committed:
Set the base theme on theme

csakiistvan committed:
Move tpl to new folder, #refactor

csakiistvan committed:
Move comment source to noew folder, #refactor

csakiistvan committed:
Move node source to new folder, #refactor

csakiistvan committed:
Delete old, add new page source, #refactor

csakiistvan committed:
Delete old theme tpl files

csakiistvan committed:
Add bootstrap theme

csakiistvan committed:
Add jQuery updtae, needs to bootstrap, #refactor

csakiistvan committed:
Move styles and images a new folder, #refactor

csakiistvan committed:
Empty file, and we not use that, #refactor

csakiistvan committed:
Fix images url, #refactor

csakiistvan committed:
Fix new scss and css folder path, #refactor

csakiistvan committed:
Colorbox is disabled, delete not used code, #refactor

csakiistvan committed:
Style secondary menu, #refactor

csakiistvan committed:
Finish header style on desktop, #refactor

csakiistvan committed:
Add other class to release output, #refactor

csakiistvan committed:
Add class to primary menu, #refactor

csakiistvan committed:
Delete breadcrumb hook, #refactor

csakiistvan committed:
Style header on mobile, #refactor

csakiistvan committed:
Style sidebar, #refactor

csakiistvan committed:
Style node on mobile, #refactor

csakiistvan committed:
Move secondary nav to better place, if we use on mobile, #refactor

csakiistvan committed:
Add more grid class to smaller screens, #refactor

csakiistvan committed:
Jump to is missing, add again, #refactor

csakiistvan committed:
More style fix on sidebar, #refactor

csakiistvan committed:
Hide source map files from repo, #refactor

csakiistvan committed:
Enable sourcemap to css, #refactor

csakiistvan committed:
Fix grey boxes position on tablet, and search form one header, #refactor

csakiistvan committed:
Fix primary menu problem on table device, #refactor

csakiistvan committed:
Use new layout on site, #refactor

csakiistvan committed:
Fix grey boxes position on tablet, and search form one header, #refactor

csakiistvan committed:
Style footer, #refactor

csakiistvan committed:
Add view rows class, #refactor

csakiistvan committed:
Style main page on mobile too, #refactor

csakiistvan committed:
Hide field if it is empty, #refactor

csakiistvan committed:
Sidebar list, and node title fix, #refactor

csakiistvan committed:
Refactor social link images to svg, #refactor

csakiistvan committed:
fix comment icon on search page

csakiistvan committed:
Refactor vote up down image, #refactor

csakiistvan committed:
Style comments, #refactor

csakiistvan committed:
Menu and footer layout fix, #refactor

csakiistvan committed:
Add update hook, to revert feature, #refactor

I use next number of hook, because the #293 has 7109 hook.

aboros commented:
Hogy kéne? Próbáljam ki lokál és kész? Vagy beholvasztjuk és teszünk rá egy teget és azt a teget kitesszük stage -re és kipróbáljuk többen? Az kéne szerintem. Valaki +1 ezzen. @snufkin @Sweetchuck ?

csakiistvan commented:
Szerintem próbáld ki lokálban, ha van valami amit javítsak megteszem, de szvsz nem vesztünk semmit ha megtageljük és kitesszük devre

csakiistvan committed:
We need enable jquery_update to bootstrap, #refactor

Sweetchuck commented:
Csak egy gondolat a `hook_update_N()` implementációkkal kapcsolatban.
A _dev_ branch-ben az utolsó: `dhu_update_7108()`
Tehát az _issue-292_ és a _theming_ branch-ben is a `dhu_update_7109()`-et kellett volna választani.
Majd a második branch beolvasztásakor a konfliktust feloldani.
Jelenleg a _theming_ branch-ben `dhu_update_7110()` van. Ami azt jelenti, hogy ha ezt a branch-et önmagában deploy-olnám az éles oldalra, szépen működne mert a drupal a 7108 után lefuttatná a 7110-et. Viszont ha utána deploy-olnám az _issue-292_ branch-et, akkor már a 7109 egyáltalán nem futna le.

Semmit nem kell változtatni a kódon mert vagy együtt fogom kirakni a 2 branch-et vagy a megfelelő sorrendben.

csakiistvan committed:
Fix the problem on header

On FF the serach submit button was a wrong place, change logic and fix the problem

csakiistvan committed:
fix perhaps scroll on mobile device from comments

csakiistvan committed:
Fix the problem on user login page, Openid field
